deploy/kubernetes/base/node/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
+- node.yaml
+
+commonLabels:
+  k8s-app: gcp-compute-persistent-disk-csi-driver
```

Comment:
the fact that we have 3 of these `kustomization.yaml` with exactly the same content seems strange. Could they be consolidated? Why are they even kustomizations and not just a part of the base yaml?